mailbox/store/src/test/java/org/apache/james/mailbox/store/TestMailboxSessionMapperFactory.java

```diff
@@ -66,17 +66,30 @@
     private static final MessageUid UID = MessageUid.of(28);
     private static final long UID_VALIDITY = 42;
 
+    private final MailboxPath mailboxPath1;
+    private final MailboxPath mailboxPath2;
+    private final MailboxPath mailboxPath3;
+    private final MailboxPath mailboxPath4;
+
     private final SimpleMailbox mailbox1;
     private final SimpleMailbox mailbox2;
     private final SimpleMailbox mailbox3;
+    private final SimpleMailbox mailbox4;
     private final List<MailboxMessage> messages = new ArrayList<MailboxMessage>();
     private final MailboxMapper mailboxMapper;
     private final MessageIdMapper messageIdMapper;
 
     public TestMailboxSessionMapperFactory() {
-        mailbox1 = new SimpleMailbox(new MailboxPath("#private", "user", "INBOX"), UID_VALIDITY, TestId.of(36));
-        mailbox2 = new SimpleMailbox(new MailboxPath("#private", "user", "OUTBOX"), UID_VALIDITY, TestId.of(46));
-        mailbox3 = new SimpleMailbox(new MailboxPath("#private", "user", "SENT"), UID_VALIDITY, TestId.of(56));
+
+        mailboxPath1 = new MailboxPath("#private", "user", "INBOX");
```

Comment:
It's bad. How do you know test will use that ?

Please introduce a MessageIdManagerFixture that holds these MailboxPaths and Mailboxes, to avoid duplication.